### PR TITLE
feat(logging): support non recording span

### PR DESCRIPTION
**Pull request by CoderPoet**
Supports logging of non-recording spans

### CODE REVIEW

**Review by GuangmingLuo — APPROVED** [review]

File: logging/zap/go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/cloudwego/kitex v0.4.2
+	github.com/stretchr/testify v1.8.0
```
Comment (GuangmingLuo) [inline logging/zap/go.mod:7]: 为啥会单独引入这个依赖呀

Comment (CoderPoet) [reply logging/zap/go.mod:7]: > 为啥会单独引入这个依赖呀

单测里用到了

Comment (li-jin-gou) [reply logging/zap/go.mod:7]: 补充licenses

Comment (CoderPoet) [reply logging/zap/go.mod:7]: > 补充licenses

license有的 https://github.com/kitex-contrib/obs-opentelemetry/blob/main/licenses/LICENSE-testify.txt

**Pull request merged**